gtsfm/view_graph_estimator/view_graph_estimator_base.py

```diff
@@ -266,6 +268,7 @@ def create_computation_graph(
                 wrapped as Delayed.
             keypoints: keypoints for each image, wrapped as Delayed.
             two_view_reports: Dict from (i1, i2) to TwoViewEstimationReport that contains metrics, wrapped as Delayed.
+            relative_pose_priors: priors on the pose between camera pairs (not delayed).
```

Comment:
maybe also add how its used: only to prune connected component.

Reply:
addressed [ddae6be](https://github.com/borglab/gtsfm/pull/569/commits/ddae6be1906674515722b8ad6a672428dd66bc20)